src/components/Loader/index.js

```diff
@@ -0,0 +1,26 @@
+import styled, { keyframes } from 'styled-components'
```

Comment:
Wooah! I didn't know SC includes a `keyframes` helper!

Reply:
It registers it globally so we could have a `/styles/animations` file for shared animations :)